Services/DeviceClient.cs

```diff
@@ -41,27 +42,54 @@ public class DeviceClient : IDeviceClient
         private readonly Azure.Devices.Client.DeviceClient client;
         private readonly ILogger log;
         private readonly IoTHubProtocol protocol;
+        private readonly IRateLimiting rateLimiting;
         private readonly string deviceId;
         private readonly IScriptInterpreter scriptInterpreter;
 
         //used to create method pointers for the device for the IoTHub to callback to
+        // ReSharper disable once NotAccessedField.Local
```

Comment:
nit: added comment from resharper?

Reply:
yes, resharper keeps warning about this var not being used (something we discussed in another PR), which was added on purpose (you did the work to support methods a while back). The comment is there so that I don't get resharper warnings

Reply:
Got it.  I will clean this up in my next PR... it's less clean to see the warnings and to have this in the repo.

Reply:
fixed in the other PR 👍